Assignment 5.Rmd

```diff
@@ -58,15 +58,17 @@ score_ctree <- ctree(factor(advice) ~ prior_prob_count + prior_percent_correct +
 plot(score_ctree)
 ```
 
-Please interpret the tree, which two behaviors do you think the teacher should most closely pay attemtion to?
+Q1: Please interpret the tree, which two behaviors do you think the teacher should most closely pay attemtion to?
+
+A1: The teacher should look into whether the student asked for hints more than 12 times. If the student asked for hints 12 times or less, they should see whether the student's prior performance was 62.9% or less.
```

Comment:
Great interpretation of the model, it would be a good idea to explain a bit more about how you reached that conclusion though.